### ISSUE TITLE
Installation fails with some modules from Xiphos

### ISSUE THREAD

**Issue by Teofilios** [crosswire/xiphos#1097]
Hello again,

There is still a problem to download/install modules from Xiphos. I have tried to install following modules without success: EarlyFathers (Books section), BibleArtBW, DoreWoodcuts and DoreWoodcutsCom (Maps and Images section). I get the message "Some of the selected works were not installed." I had also problem with other modules, but managed to install them after I have repeatedly trying to install them three or four times.

I was using BibleTimes 3.0.2 in Linux Debian 11.

Best regards,
Aleksandar

**Comment by LAfricain** [comment]
I'm not sure it's a xiphos but, it's only the xiphos server. I have the same issue. @karlkleinpaste any idea?

**Comment by karlkleinpaste** [comment]
the xiphos ftp machine serves ~15Gbytes/day all over the planet. it's healthy and shipping stuff out as i write. i have a VNC session on it in which i have a terminal tail'ing xferlog. i don't know.

well, ok, it's even busier than usual lately. from last night's logwatch email summary:
```
--------------------- ftpd-xferlog Begin ------------------------ 
 TOTAL KB OUT: 20642278KB (20642MB)
```
no one else is reporting any trouble.

all that said, xferlog indicates that some things are taking much longer to transfer than they seemingly should. i may reboot the box just to give it a fresh start. it's been up a long time. i need to wait for an apparent quiescent moment so i don't interrupt someone's in-progress install.

**Comment by Teofilios** [comment]
Thanks for following up this iisue. I will try to install a little bit later.

**Comment by Teofilios** [comment]
I have tried once again to install the moduls in question without success. I will try tomorrow and if I manage to install some of these, I will let you know.

**Comment by Teofilios** [comment]
I have tried several times to install the modules in question with no luck. Very odd. I have managed to install all other modules from other servers (e.g., CrossWire), but expereinced much trouble to install from Xiphos.

**Comment by LAfricain** [comment]
> no one else is reporting any trouble.

Yes Karl Me too I can't install from xiphos.

**Comment by karlkleinpaste** [comment]
try an installation using the command line tool `installmgr`.

there continues to be a great deal of very ordinary traffic emanating from the host, including to xiphos users, as indicated by xferlog.

**Comment by karlkleinpaste** [comment]
nothing further said, and the server is shoveling data at a typically impressive rate over the last week, auto-summarized nightly:
 TOTAL KB OUT: 17084404KB (17084MB)
 TOTAL KB OUT: 17044414KB (17044MB)
 TOTAL KB OUT: 15171815KB (15171MB)
 TOTAL KB OUT: 19451232KB (19451MB)
 TOTAL KB OUT: 14276735KB (14276MB)
 TOTAL KB OUT: 18963371KB (18963MB)
 TOTAL KB OUT: 16744015KB (16744MB)
 TOTAL KB OUT: 12054779KB (12054MB)
nothing wrong here.

**Comment by CataclysmicStorm** [comment]
Hello, I am having a similar issue with all module repositories, anytime the download is a few MB big, it goes to downloading 3/6 and goes slowly, then suddenly says "install failed" I tried to fiddle with the timeoutminni or whatever it is in conf but it didnt help. this is an issue with the timeout for download times I believe, as I have been able to install smaller modules just fine, and I can manually install modules (I just have to find them on the various websites) sadly a few modules, such as John Gill's commentaries are only downloadable through the module manager and cannot be found on Xiphos's website.

![Screenshot_20250102_205643](https://github.com/user-attachments/assets/3153f89c-391c-4e7e-8f84-8272c3d0d61f)

what do i do?
